source/elements/oneDAL/source/data_management/accessors.rst

```diff
@@ -50,40 +43,32 @@ Accessor Types
 --------------
 
 |dal_short_name| defines a set of accessor classes. Each class is associated
-with a single specific way of interacting with data within a :txtref:`dataset`.
-The following table briefly explains these classes and shows which
-:txtref:`dataset` implementations are supported by each accessor type.
+with a single specific way of interacting with data within a :txtref:`table`.
```

Comment:
Each class supports one specific way of reading/extracting/... data from the ...